src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LogValuesFormatter.cs

```diff
@@ -20,6 +20,10 @@ internal class LogValuesFormatter
         private readonly string _format;
         private readonly List<string> _valueNames = new List<string>();
 
+        // NOTE: If this assembly ever builds for netcoreapp, the below code should change to:
```

Comment:
Doesn't it cross compile?

Reply:
No - a lot of the Extensions assemblies don't build for netcoreapp.

https://github.com/dotnet/runtime/blob/3c801a8a3b29eabc29ae88e1961c772fda17a1d2/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/Microsoft.Extensions.Logging.Abstractions.csproj#L4

Reply:
not sure if that matters but that would change as we enable nullables on the M.E. projects. e.g. [csproj diff here](https://github.com/dotnet/runtime/pull/43892/files?file-filters%5B%5D=.csproj#diff-b0675ddd797a8ed03ce040c5f2573e0817fd2a7bfa2a940de38b3c21187b7a0d)